reference/robot.md

```diff
@@ -694,6 +696,8 @@ The message is received using the `webots.window("<robot window name>").receive`
 
 **wb\_robot\_window\_custom\_function** - *communication with the native C/C++ robot window [deprecated]*
 
+{[C++](cpp-api.md#cpp_robot)}, {[Java](java-api.md#java_robot)}, {[Python](python-api.md#python_robot)}, {[Matlab](matlab-api.md#matlab_robot)}, {[ROS](ros-api.md)}
+
```

Comment:
Why not using `ros-api.md#robot` like for other languages?

Reply:
Probably the tags in the ROS api page have been added later after this was set.
All the references to the ROS API in any node section doesn't use the node tag.
We should open an issue for this.